pkg/karmadactl/top/top.go

```diff
@@ -0,0 +1,58 @@
+package top
+
+import (
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+	metricsapi "k8s.io/metrics/pkg/apis/metrics"
+)
+
+const (
+	sortByCPU    = "cpu"
+	sortByMemory = "memory"
+)
+
+var (
+	supportedMetricsAPIVersions = []string{
+		"v1beta1",
+	}
+	topLong = templates.LongDesc(i18n.T(`
+		Display Resource (CPU/Memory) usage of member clusters.
+
+		The top command allows you to see the resource consumption for nodes or pods of member clusters.
+
+		This command requires Metrics Server to be correctly configured and working on the server. `))
+)
+
+func NewCmdTop(f cmdutil.Factory, parentCommand string, streams genericclioptions.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "top",
+		Short: i18n.T("Display resource (CPU/memory) usage of member clusters"),
```

Comment:
kubectl i18n package cannot be used by karmadactl, so it's better to remove this line.

Reply:
Updated.